overlord/healthstate/healthstate_test.go

```diff
@@ -98,7 +98,6 @@ const (
 	noHook = iota
 	badHook
 	goodHook
-	captainHook
```

Comment:
heh, a little easter egg

Reply:
The rest of 3dfbe52e1217f82b62fce0b422398d6a979ce7cd is also very funny, I don't know why it's comited under my name, this was certainly John :)

Also sightly sad that we remove these Easter eggs but maybe it's part of "growing up" (we used to have e.g. snap blame)

Reply:
`snap changes everything` ;)